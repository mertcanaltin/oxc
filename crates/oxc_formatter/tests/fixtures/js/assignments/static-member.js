options = BracketMatchingControllerLONGLONGLONG._DECORATION_OPTIONS_WITHOUT_OVERVIEW_RULER;
