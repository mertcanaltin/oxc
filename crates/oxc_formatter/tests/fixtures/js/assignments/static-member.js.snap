---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
options = BracketMatchingControllerLONGLONGLONG._DECORATION_OPTIONS_WITHOUT_OVERVIEW_RULER;

==================== Output ====================
options =
  BracketMatchingControllerLONGLONGLONG._DECORATION_OPTIONS_WITHOUT_OVERVIEW_RULER;

===================== End =====================
